.github/issue_template.md

```diff
@@ -11,7 +19,9 @@ WDL https://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team
 CWL https://www.biostars.org/
 -->
 
-<!-- Are you seeing something that looks like a bug?  Then great!  You're in the right place.  -->
+<!-- Are you seeing something that looks like a bug?  Then great!  You're almost in the right place.  -->
+
+<!-- You'll want to go to https://broadworkbench.atlassian.net/projects/BA/issues and then tell us: -->
```

Comment:
We should probably remove all this "remember to include" cruft while we're here. We can always make them JIRA fields instead

Reply:
Have a link?

Briefest of googling, I only found a way to do templates at ~$10 per Jira user.

Reply:
🤢

Reply:
QA has it set up for prod issues, so if they can do I'm guessing we can too